tenable_io/api/sc_containers.py

```diff
@@ -0,0 +1,28 @@
+from tenable_io.api.base import BaseApi
+from tenable_io.api.models import ScContainer
+
+
+class ScContainersApi(BaseApi):
+
+    def delete(self, repository_name, sha256):
+        """Delete a container image.
+
+        :param repository_name: The registry/path of the image. For example, if you have a repository named unix and an
+            image named alpine, you would type unix/alpine.
+        :param sha256: SHA256 checksum of the container image.
+        :raise TenableIOApiException:  When API error is encountered.
+        :return: True if successful.
+        """
+        uri = 'container-security/api/v1/container/%(repository_name)s/manifests/%%(sha256)s' % \
```

Comment:
maybe this for consistency? 
`uri = 'container-security/api/v1/container/%(repository_name)s/manifests/%(sha256)s'
        self._client.delete(uri, path_params={'repository_name': repository_name, 'sha256': sha256})`